src/code42cli/click_ext/groups.py

```diff
@@ -19,6 +23,26 @@
 _DIFFLIB_CUT_OFF = 0.6
 
 
+class InterpreterGroup(click.Group):
+    """A `click.Group` subclass that captures the `--script` option and short-circuits
+    the normal CLI parsing, passing all subsequent args to the python interpreter that
+    the CLI is installed into, allows for reliably executing custom scripts that import
+    code42cli.
+    """
+
+    def parse_args(self, ctx, args):
+        if len(args) > 1 and args[0] == "--script":
+            args[0] = sys.executable
+            if platform.system() == "Windows":
```

Comment:
Can we sanitize the args somehow?
Probably a far-fetched concern, but if the CLI was running on a server, like maybe as part of an integration of some sort, and users were able to call custom scripts remotely, could this could be used as part of a shell-injection attack?

Bandit scan:
```
 Issue: [B605:start_process_with_a_shell] Starting a process with a shell, possible injection detected, security issue.
   Severity: High   Confidence: High
   Location: src/code42cli/click_ext/groups.py:40
   More Info: https://bandit.readthedocs.io/en/latest/plugins/b605_start_process_with_a_shell.html
39                      cmd = shlex.join(args)
40                  status = os.system(cmd)
41                  sys.exit(status)
```

Oddly, `bandit` claims it is a HIGH severity but the documentation says it should be LOW/

Reply:
I don't think we need to worry about that since this function is explicitly _for_ running user-provided code. If someone can hijack the actual execution of the CLI to get it to call `--script` on something unintended, they likely already have the ability to just run any arbitrary command they want.

Reply:
I was imagining a scenario where some integration or app intentionally allowing user input to put into the CLI command, which like I said, is maybe a far-fetched scenario. But, in this scenario, they expose the CLI essentially, so it's not hijacked (yet).

Reply:
but it would be outside of our control if someone did that, albeit a dangerous decision on their part, but sanitization on our end could be an extra layer of security

Reply:
I did start out playing around with just having `code42 --python` just return the full path to the interpreter `code42` was installed under. So then if a user was having problems with the regular `python script.py` they could try `$(code42 --python) script.py`.

Reply:
I think I'll just revert the exec python logic back to just printing the python path. It keeps the code simple and gives users a relatively easy path to figuring things out if their environment is broken.